src/Sentry.Unity/Integrations/SceneManagerIntegration.cs

```diff
@@ -32,15 +32,15 @@ void OnSceneManagerOnSceneLoaded(SceneAdapter scene, LoadSceneMode mode)
 
                 hub.AddBreadcrumb(
                     $"Scene '{scene.Name}' was loaded",
-                    category: "scene.loaded",
+                    category: "scene.loaded"
                     // TODO: What is worth paying the price of allocation in order to add here?
-                    data: new Dictionary<string, string>
-                    {
-                        {"name", scene.Name},
+                    // data: new Dictionary<string, string>
```

Comment:
As agreed, lets drop all this dead code and avoid allocating the dictionary to begin with.